packages/databricks-vscode/resources/python/00-databricks-init.py

```diff
@@ -403,18 +405,24 @@ def init_ui(self):
         def update_ticks(
             self,
             stages,
-            inflight_tasks: int
+            inflight_tasks: int,
+            done: bool
         ) -> None:
-            total_tasks = sum(map(lambda x: x.num_tasks, stages))
-            completed_tasks = sum(map(lambda x: x.num_completed_tasks, stages))
+            total_tasks = builtinSum(map(lambda x: x.num_tasks, stages))
```

Comment:
Just for general understanding - what was wrong with the old approach here?

Reply:
I ran into a case where sum imported from pyskaprk.sql.functions was shadowing the builtin. I've added a comment.